doc/configuration.rst

```diff
@@ -184,6 +185,31 @@ As the patterns are parsed as `regular expressions`_, users are advised to consu
 
         $ sphinx-build -D sphinx_gallery_conf.filename_pattern=plot_specific_example\.py ...
 
+.. _reset_argv:
+
+Passing command line arguments to example scripts
+=================================================
+
+By default, Sphinx-Gallery will not pass any command line arguments to example scripts.
+By setting the ``reset_argv`` option, it is possible to change this behavior and pass command line arguments to example scripts.
+``reset_argv`` needs to be a Callable that accepts the ``gallery_conf`` and ``script_vars`` dictionaries and returns a list of strings that are passed as additional command line arguments to the interpreter.
```

Comment:
Nitpick:
```suggestion
``reset_argv`` needs to be a Callable that accepts the ``gallery_conf`` and ``script_vars`` dictionaries as input and returns a list of strings that are passed as additional command line arguments to the interpreter.
```

also this line seems longer than 80 characters?

Reply:
Thanks, updated in fcf0db3ba3253cf0e0b793a2dd4f395296d9a968